cmd/daemon/contract_test.go

```diff
@@ -1,32 +1,28 @@
 package main
 
 import (
-	"math/big"
 	"testing"
 
 	"github.com/noot/atomic-swap/common"
 	"github.com/noot/atomic-swap/tests"
 
 	ethcommon "github.com/ethereum/go-ethereum/common"
 	ethcrypto "github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/stretchr/testify/require"
 )
 
 func TestGetOrDeploySwapFactory(t *testing.T) {
 	pk, err := ethcrypto.HexToECDSA(tests.GetTakerTestKey(t))
 	require.NoError(t, err)
 
-	ec, err := ethclient.Dial(common.DefaultEthEndpoint)
```

Comment:
I created the `tests.NewEthClient` function to eliminate some boilerplate code and reduce the code points that need changing if you want to test on a different chain.